.devcontainer/Dockerfile

```diff
@@ -1,9 +1,11 @@
-# See here for image contents: https://github.com/openinf/docker-fish
```

Comment:
I did not mean to change this, but this wasn't too bad of a devcontainer (took forever to load).